WordPress/Classes/Utility/BuildInformation/FeatureFlag.swift

```diff
@@ -87,6 +90,7 @@ extension FeatureFlag {
         case .tipKit: "TipKit"
         case .sidebar: "Sidebar"
         case .newGutenberg: "Experimental Block Editor"
+        case .newGutenbergThemeStyles: "Experimental Block Editor Styles"
```

Comment:
Added to test the functionality, ideally, this should be an app setting that can be synced